src/Tools/dotnet-monitor/Startup.cs

```diff
@@ -33,24 +34,32 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc(options =>
-                {
-                    options.Filters.Add(new ProducesAttribute("application/json"));
-
-                    options.EnableEndpointRouting = false;
-                })
-                .SetCompatibilityVersion(CompatibilityVersion.Latest)
-                .AddApplicationPart(typeof(DiagController).Assembly);
+            // AddControllers is sufficient because the tool does not use Razor nor Views.
+            services.AddControllers(options =>
```

Comment:
We disable endpoint routing here but enable it later on. Is this intended?

Reply:
It was there before, but doesn't seem to be necessary.